sample/desktopApp/src/jvmMain/kotlin/main.kt

```diff
@@ -32,7 +33,12 @@ fun main() =
                                 initialized = true
                             }
                         }
-                        release("jbr-release-17.0.10b1087.23")
+                        download(
+                            Download.Builder().github {
+                                release("jbr-release-17.0.10b1087.23")
```

Comment:
This version wont work anymore I guess.

The KCEF release tag is aligned to the date when jcef was updated.
In this case `20 April 2024`. (They updated some ID from `String` to `Long` if I remember correctly)

So probably [Release 17.0.10b1207.14](https://github.com/JetBrains/JetBrainsRuntime/releases/tag/jbr-release-17.0.10b1207.14) or [Release 17.0.11b1207.20](https://github.com/JetBrains/JetBrainsRuntime/releases/tag/jbr-release-17.0.11b1207.20) will work, but I haven't tested it.

Additionally a cleaner way that should work too would be:
```kotlin
download {
    release("<version>")
}
```

Reply:
Oh it must have been cached when I tested it, I'll try one of those